Common/File/DiskFree.cpp

```diff
@@ -43,7 +43,7 @@ bool free_disk_space(const Path &path, uint64_t &space) {
 	if (res == 0) {
 #ifndef __ANDROID__
 		if (diskstat.f_flag & ST_RDONLY) {
-			space = 0;
+			space = -1;
 			return true;
```

Comment:
It returns true though, isn't it better to interpret that as 0 bytes free?  We'll cast this to unsigned and try to write.

Otherwise it should at least return false?  But this isn't really an error.  The free space for a game to create new save data is 0 bytes.  It's a known value, so I'd consider it successful.

-[Unknown]